core/src/main/java/com/novoda/noplayer/exoplayer/InfoListener.java

```diff
@@ -1,30 +1,12 @@
 package com.novoda.noplayer.exoplayer;
 
-import com.google.android.exoplayer2.Format;
-import com.google.android.exoplayer2.upstream.DataSpec;
-
-import java.io.IOException;
+import java.util.Map;
 
 /**
  * A listener for debugging information.
  */
 public interface InfoListener {
 
-    void onDroppedFrames(int count, long elapsed);
```

Comment:
What's the reason behind this change on this interface? Didnt' see it in the diff